CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## How to Contribute to OpenMM Development
+
+We welcome anyone who wants to contribute to the project, whether by adding a feature,
+fixing a bug, or improving documentation.  The process is quite simple.
+
```

Comment:
Would it make sense to add some structure within this document, like
```
### General process 
...
### Submitting a feature
...
### Fixing a bug
...
### Improving documentation
...
```

Reply:
I'm not sure what we'd say in the different cases.  The process is pretty much the same regardless: open an issue to discuss it, then submit a PR with the changes.  And since the whole document is only a few paragraphs long, it doesn't really need a lot of organization.